components/brave_wallet_ui/utils/datetime-utils.ts

```diff
@@ -67,16 +67,15 @@ export function serializedTimeDeltaToJSDate(
   return new Date(Number(timeDelta.microseconds) / 1000)
 }
 
+const dateTimeFormatter = new Intl.DateTimeFormat(undefined, {
+  hour12: true,
```

Comment:
Ideally we should use the value that's set by the platform/browser. i.e. 24h time on `fr_CA` and 12h time on `en_US`.

Reply:
fixed: 9662645